packages/codeeditor/src/editor.ts

```diff
@@ -602,6 +612,8 @@ namespace CodeEditor {
    */
   export
   let defaultConfig: IConfig = {
+    fontFamily: '\'Source Code Pro\', monospace',
+    fontSize: 13,
```

Comment:
The other thing that contributes significantly to code readability is `line-height`... this has a variable, `--jp-code-line-height`, and is particularly fickle with respect to which font a user might pick.